specification/cognitiveservices/data-plane/WebSearch/v1.0/WebSearch.json

```diff
@@ -0,0 +1,1235 @@
+      ]
+    }
+  },
+  "host": "api.cognitive.microsoft.com",
+  "schemes": [
+    "https"
+  ],
+  "basePath": "/bing/v7.0",
+  "produces": [
+    "application/json"
+  ],
+  "securityDefinitions": {
+    "apiKeyHeader": {
+      "name": "Ocp-Apim-Subscription-Key",
+      "type": "apiKey",
+      "in": "header"
+    },
+    "apiKeyQuery": {
+      "name": "subscription-key",
+      "type": "apiKey",
+      "in": "query"
+    }
+  },
+  "security": [
+    {
+      "apiKeyHeader": []
+    },
+    {
+      "apiKeyQuery": []
+    }
+  ],
+  "paths": {
+    "/search": {
+      "get": {
+        "summary": "The Web Search API lets you send a search query to Bing and get back search results that include links to webpages, images, and more.",
+        "operationId": "Web_Search",
+        "tags": [
+          "WebSearch"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/x-bingapis-sdk"
+          },
+          {
+            "name": "Accept",
+            "in": "header",
+            "description": "The default media type is application/json. To specify that the response use [JSON-LD](http://json-ld.org/), set the Accept header to application/ld+json.",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "Accept-Language",
+            "x-ms-client-name": "AcceptLanguage",
+            "in": "header",
+            "description": "A comma-delimited list of one or more languages to use for user interface strings. The list is in decreasing order of preference. For additional information, including expected format, see [RFC2616](http://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html). This header and the setLang query parameter are mutually exclusive; do not specify both. If you set this header, you must also specify the cc query parameter. Bing will use the first supported language it finds from the list, and combine that language with the cc parameter value to determine the market to return results for. If the list does not include a supported language, Bing will find the closest language and market that supports the request, and may use an aggregated or default market for the results instead of a specified one. You should use this header and the cc query parameter only if you specify multiple languages; otherwise, you should use the mkt and setLang query parameters. A user interface string is a string that's used as a label in a user interface. There are very few user interface strings in the JSON response objects. Any links in the response objects to Bing.com properties will apply the specified language.",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "Pragma",
+            "in": "header",
+            "description": "By default, Bing returns cached content, if available. To prevent Bing from returning cached content, set the Pragma header to no-cache (for example, Pragma: no-cache).",
```

Comment:
Please consider using the default property to document default values rather than using the description to do so...

Reply:
As per my discussion with Garrett, we would like to go ahead with this for now. I am tracking these expected changes as a bug. 
https://msasg.visualstudio.com/Bing_UX/DevEx%20Team/_workitems?id=966145&_a=edit